package.json

```diff
@@ -25,9 +25,10 @@
   },
   "private": true,
   "scripts": {
-    "build": "lerna run prepack",
+    "build": "lerna run prepack --concurrency 4",
     "test": "lerna run test --concurrency 4",
     "version": "cp packages/cli/CHANGELOG.md CHANGELOG.md && git add CHANGELOG.md",
+    "cleanManifests": "rm -rf ./packages/*/oclif.manifest.json",
```

Comment:
I was curious as to what function this script serves? Does this serve as clean up after generating  the manifest file after running tests?

Reply:
Good question. The manifest files are generated so oclif can understand what commands are available. Without them  oclif/test.command will load and transpile all plugins on every call.
This was helpful while I was debugging, but isn't necessary anymore since I added these files to .gitignore.